src/Lib/Canvas.vala

```diff
@@ -339,24 +361,29 @@ public class Akira.Lib.Canvas : Goo.Canvas {
             return;
         }
 
-        double x, y;
-        target.get ("x", out x, "y", out y);
+        double x, y, width, height;
+        target.get ("x", out x, "y", out y, "width", out width, "height", out height);
 
         var item = (target as Goo.CanvasItemSimple);
 
-        var line_width = 2.0 / get_scale ();
-        var stroke = item.line_width;
-        var real_x = x - (line_width * 2);
-        var real_y = y - (line_width * 2);
-        var width = item.bounds.x2 - item.bounds.x1 + stroke - line_width;
-        var height = item.bounds.y2 - item.bounds.y1 + stroke - line_width;
+        var line_width = get_scale() * 2;
```

Comment:
the division might be correct as it was before. When zooming in the hover effect line is now thicker than before